cdap-ui/app/cdap/components/PipelineDetails/RunLevelInfo/RunConfigs.js

```diff
@@ -67,10 +67,12 @@ export default class RunConfigs extends Component {
   }
 
   getRuntimeArgsAndToggleModeless = () => {
-    PipelineConfigurationsStore.dispatch({
-      type: PipelineConfigurationsActions.SET_MODELESS_OPEN_STATUS,
-      payload: { open: false },
-    });
+    if (this.state.showModeless) {
+      PipelineConfigurationsStore.dispatch({
+        type: PipelineConfigurationsActions.SET_MODELESS_OPEN_STATUS,
+        payload: { open: false },
+      });
+    }
```

Comment:
if we are closing the modeless here, is there a need to fetch the runtime args?

Reply:
So we are closing the pipeline config modeless from here because we only want one to be open (because they are using a component that uses same store). So this will make sure we only close the pipeline config modeless when we open runtime arguments at runlevel.